x-pack/solutions/search/plugins/search_getting_started/public/components/tutorials/console_tutorials_group.tsx

```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { consoleTutorials } from '@kbn/search-code-examples';
+import { TryInConsoleButton } from '@kbn/try-in-console';
+import { EuiCard, EuiFlexGroup, EuiFlexItem, EuiText, EuiImage } from '@elastic/eui';
+import React, { useRef } from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { useKibana } from '../../hooks/use_kibana';
+import { SearchGettingStartedSectionHeading } from '../section_heading';
+import searchWindowIllustration from '../../assets/search_window_illustration.svg';
+import searchResultsIllustration from '../../assets/search_results_illustration.svg';
+import searchObserveIllustration from '../../assets/search_observe_illustration.svg';
+import commandLineIllustration from '../../assets/command_line.svg';
+interface TutorialMetadata {
+  title: string;
+  dataTestSubj: string;
+  description: string;
+  request: string;
+  image: string;
+  buttonRef: React.RefObject<HTMLButtonElement>;
+}
+
+export const ConsoleTutorialsGroup = () => {
+  const { application, console: consolePlugin, share } = useKibana().services;
+  const tutorials: TutorialMetadata[] = [
```

Comment:
Not a huge concern but having these in function scope means they get re-created on every render. Which might be needed for the `useRef` calls, but gives a code smell to me. It's probably fine though?